### PR TITLE
Update index.html.erb

### PR DESCRIPTION
**Pull request by jeremysb1**
This is to put the Startup Institute in the Scholarship section on the code schools page. This should close issue #424.

### CODE REVIEW

File: app/views/code_schools/index.html.erb [hunk 1]
```diff
@@ -60,9 +60,13 @@
...
-
-        <a href="http://go.flatironschool.com/operation-code-overview" target="_blank">Learn more...</a></p>
+        <p>New York City-based, Flatiron School provides up to $25,000 in scholarships for Operation Code veterans to participate in the Learn-Verified program which includes, $500 off the first month's tuition for the first 50 eligible applicants. Not sure about committing full-time? Sign up for  <a href="https://learn.co/join/operation-code" target="_blank">Learn.co</a>, an online platform to get introduced to web development and Ruby on Rails. If you get stuck, join the <a href="https://operation-code.slack.com/messages/general/#learn-dot-co" target="_blank"> #learn-dot-co</a> channel in our Slack. <a href="http://go.flatironschool.com/operation-code-overview" target="_blank">Learn more...</a></p>
```
Comment (hollomancer) [inline app/views/code_schools/index.html.erb]: Any reason why "Learn more..." ended up on a single line here?

File: app/views/code_schools/index.html.erb [hunk 2]
```diff
@@ -60,9 +60,13 @@
...
+        
+      <hr>
+      
+      <h5><a href="https://www.startupinstitute.com/" target="_blank">Startup Institute (Boston and NYC)</a></h5>
+        <p>Startup Institute offers all veterans an <a href="https://www.startupinstitute.com/admissions/scholarships/" target="_blank">automatic $1,000 off</a>the cost of their full-time program. Train to work in the innovation economy in the areas of web development, web design, digital marketing, or sales and account management. Additional scholarships are also available for: Women in Tech, People of Color in Tech, Seasoned Professional in Tech (age 40+), and Excellence in STEM. <a> href="https://www.startupinstitute.com/admissions/scholarships/" target="_blank">Learn more...</a></p>
```
Comment (hollomancer) [inline app/views/code_schools/index.html.erb]: Same comment here.

File: app/views/code_schools/index.html.erb (hunk 1 above)
Comment (jeremysb1) [inline app/views/code_schools/index.html.erb]: no idea.

File: app/views/code_schools/index.html.erb (hunk 2 above)
Comment (jeremysb1) [inline app/views/code_schools/index.html.erb]: so do we want learn more on a separate line?

File: app/views/code_schools/index.html.erb (hunk 1 above)
Comment (jeremysb1) [inline app/views/code_schools/index.html.erb]: i didn't write that in for flatiron school

**Pull request merged**